generator.sh

```diff
@@ -28,12 +28,59 @@ read hostname
 echo "What Bundle Identifier would you like to use? (ex: com.yourcompany.yourapp)"
 read bundle_identifier
 
-mkdir $project_name
-cd $project_name
+ios_ci_support=0
+android_ci_support=0
+
+read -p "Do you want CircleCI support? (y/n) " -n 1 -r
+echo
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
+    # if yes, inform user of Circle setup document
+    read -p "Do you want iOS CI support? (y/n) " -n 1 -r
```

Comment:
Good point - I thought it would be easier just to nuke the circle folder if you got an initial no, but that could still be done if you say no to both questions. I'll update it!